ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -165,6 +165,11 @@ public void updateExtraData(ReactEditText view, Object extraData) {
     }
   }
 
+  @ReactProp(name = ViewProps.COLOR, defaultInt = 0, customType="color")
```

Comment:
@dmmiller If i set the color and then rerender without color, it fails with "Error while updating property 'color' of a view managed by: AndroidTextInput"